packages/wrangler/src/inspect.ts

```diff
@@ -744,8 +838,15 @@ function logConsoleMessage(evt: Protocol.Runtime.ConsoleAPICalledEvent): void {
 /**
  * Opens the chrome debugger
  */
-export const openInspector = async (inspectorPort: number) => {
-	const url = `https://devtools.devprod.cloudflare.dev/js_app?theme=systemPreferred&ws=localhost:${inspectorPort}/ws`;
+export const openInspector = async (
+	inspectorPort: number,
+	worker: string | undefined
+) => {
+	const query = new URLSearchParams();
+	query.set("theme", "systemPreferred");
+	query.set("ws", `localhost:${inspectorPort}/ws`);
+	if (worker) query.set("domain", worker);
```

Comment:
`domain` is a custom query parameter to override the origin shown in DevTools (one of the patches to DevTools). It was implemented back in February to provide a nicer experience in the Sources panel, which would otherwise show a Worker's sources under a folder called `(no domain)`